ballista/rust/core/proto/ballista.proto

```diff
@@ -424,54 +424,87 @@ message ExecutionGraph {
   uint64 output_partitions = 5;
   repeated PartitionLocation output_locations = 6;
   string scheduler_id = 7;
+  uint32 tid_gen = 8;
```

Comment:
nit: `tid_gen` isn't a descriptive name. I am guessing this is short for `task_id_gen`?

Reply:
Sure,  will change to `task_id_gen`